### PR TITLE
translateionRunner: improve minimalist options

### PR DESCRIPTION
**Pull request by yookoala**
* Properly added flags.
* Added alias: -h, -f

### CODE REVIEW

**Review by ykzts — APPROVED** [review]

File: config/webpack/translationRunner.js [hunk 1]
```diff
@@ -14,7 +23,7 @@ const availableLanguages = fs.readdirSync(`${process.cwd()}/${translationsDirect
 }, []);
 
 // print help message
-if (argv.help !== undefined) {
+if (argv.help === true) {
```
Comment (ykzts) [inline config/webpack/translationRunner.js]: Is not it "`if (argv.help)`"?

File: config/webpack/translationRunner.js [hunk 2]
```diff
@@ -1,30 +1,59 @@
...
-const translationsDirectory = 'app/javascript/mastodon/locales';
-const localeFn = /^([a-z]{2,3}(|\-[A-Z]+))\.json$/;
-const reRFC5646 = /^[a-z]{2,3}(|\-[A-Z]+)$/;
-const availableLanguages = fs.readdirSync(`${process.cwd()}/${translationsDirectory}`).reduce((acc, fn) => {
-  if (fn.match(localeFn)) {
-    acc.push(fn.replace(localeFn, '$1'));
+const testRFC5626 = function (reRFC5646) {
+  return function (language) {
+    if (!language.match(reRFC5646)) {
+      throw 'Not RFC5626 name';
```
Comment (ykzts) [inline config/webpack/translationRunner.js]: replace to `throw new Error('Not RFC5626 name');`.

File: config/webpack/translationRunner.js [hunk 3]
```diff
@@ -1,30 +1,59 @@
...
-  return acc;
-}, []);
+}
 
-// print help message
-if (argv.help !== undefined) {
+const testAvailability = function (availableLanguages) {
+  return function (language) {
+    if ((argv.force !== true) && availableLanguages.indexOf(language) < 0) {
+      throw 'Not an available language';
```
Comment (ykzts) [inline config/webpack/translationRunner.js]: replace to `throw new Error('Not an available language');`

**Review by ykzts — CHANGES_REQUESTED** [review]

File: config/webpack/translationRunner.js [hunk 4]
```diff
@@ -33,48 +62,45 @@ to input multiple languages, separate them with space.
...
+const reRFC5646 = /^[a-z]{2,3}(|\-[A-Z]+)$/;
+const availableLanguages = fs.readdirSync(`${process.cwd()}/${translationsDirectory}`).reduce((acc, fn) => {
+  if (fn.match(localeFn)) {
+    acc.push(fn.replace(localeFn, '$1'));
+  }
+  return acc;
+}, []);
+
+// print help message
+if (argv.help === true) {
```
Comment (ykzts) [inline config/webpack/translationRunner.js]: Is not it "`if (argv.help)`"?

**Review by ykzts — CHANGES_REQUESTED** [review]

File: config/webpack/translationRunner.js (hunk 4 above)
Comment (yookoala) [inline config/webpack/translationRunner.js]: This is more resilient to changes.

File: config/webpack/translationRunner.js (hunk 3 above)
Comment (yookoala) [inline config/webpack/translationRunner.js]: Updated. See 2d4b381

File: config/webpack/translationRunner.js (hunk 2 above)
Comment (yookoala) [inline config/webpack/translationRunner.js]: Updated. See 2d4b381

File: config/webpack/translationRunner.js (hunk 4 above)
Comment (yookoala) [inline config/webpack/translationRunner.js]: Updated. See 0292e39.

**Review by ykzts — APPROVED** [review]

**Pull request merged**